<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KitsLista</class>
 <widget class="QDialog" name="KitsLista">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista de kits</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Kits</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableKits">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Filtrar por</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Filtro</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbFiltro"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edtFiltro"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Detalles</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableKitsDetalles">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbFiltro</tabstop>
  <tabstop>edtFiltro</tabstop>
  <tabstop>tableKits</tabstop>
  <tabstop>tableKitsDetalles</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
